mantle/kola/tests/util/rpmostree.go

```diff
@@ -54,7 +61,27 @@ type simplifiedRpmOstreeStatus struct {
 // a limited representation of the output of `rpm-ostree status --json`
 func GetRpmOstreeStatusJSON(c cluster.TestCluster, m platform.Machine) (simplifiedRpmOstreeStatus, error) {
 	target := simplifiedRpmOstreeStatus{}
-	rpmOstreeJSON := c.MustSSH(m, "rpm-ostree status --json")
+	// We have a case where the rpm-ostree status command is failing
+	// for the ostree.hotfix test and we don't know why:
+	// https://github.com/coreos/fedora-coreos-tracker/issues/942
+	// Let's run it once and check the failure. If it fails we will
+	// always return a failure because we want to know, but we will
+	// also run it with some retries to see if it succeeds in a
+	// successive try for some reason or if it continues to fail.
+	rpmOstreeJSON, err := c.SSH(m, "rpm-ostree status --json")
+	if err != nil {
+		retryStatus := func() error {
+			_, err := c.SSH(m, "rpm-ostree status --json")
+			return err
+		}
+		err := util.Retry(10, 10*time.Second, retryStatus)
+		if err != nil {
+			plog.Errorf("rpm-ostree status failed even after retries: %v", err)
+		} else {
+			plog.Warningf("rpm-ostree status succeeded after retries.")
+		}
+		return target, err // the original error
```

Comment:
If the original invocation fails and the retries pass, we will return `nil` for the error here. If we want to return the original error I think the output from `util.Retry` should be renamed to a different variable like `err2`.